tests/Traits/EntityLoadingTrait.php

```diff
@@ -26,11 +27,24 @@ trait EntityLoadingTrait {
    *   Entity object, if any.
    */
   protected function loadEntityByLabel(string $entity_type, string $label, string $bundle = NULL): EntityInterface {
+    $exception_message = "No '$entity_type' entity of type '$bundle' with label '$label' has been found.";
     $storage = \Drupal::entityTypeManager()->getStorage($entity_type);
     $label_key = $storage->getEntityType()->getKey('label');
-    $properties = [
-      $label_key => $label,
-    ];
+    if (!$label_key) {
+      // Find entity in content storage if entity doesn't support labels.
```

Comment:
As for me it looks like a workaround and for now is useful for Behat tests only.